websites/docs-smoke-test/src/content/components/rss-feeds.mdx

```diff
@@ -8,4 +8,21 @@ import { RssFeeds } from '@commercetools-docs/ui-kit'
 
 <Info>The RSS feed is only generated in production builds so this test fails in development mode.</Info>
 
-<RssFeeds url="/docs-smoke-test/releases/feed.xml" />
+## Table with one feed
+
+<RssFeeds dataSources={
+  `[
+    {"title": "Merchant Center", "url": "https://docs.commercetools.com/merchant-center/releases/feed.xml"}
+  ]`
+}/>
+
+## Table with multiple feeds
+
+<RssFeeds dataSources={
+  `[
+    {"title": "HTTP API", "url": "https://docs.commercetools.com/api/releases/feed.xml"},
+    {"title": "Merchant Center", "url": "https://docs.commercetools.com/merchant-center/releases/feed.xml"},
+    {"title": "Custom Applications", "url": "https://docs.commercetools.com/custom-applications/releases/feed.xml"},
+    {"title": "Import API", "url": "https://docs.commercetools.com/import-api/releases/feed.xml"}
+  ]`
+}/>
```

Comment:
We should pass a list, not a string.

Also, I think that the title can be extracted from the RSS feed, so we also don't need to explicitly pass it.

```suggestion
<RssFeeds dataSources={[
    "https://docs.commercetools.com/api/releases/feed.xml",
    "https://docs.commercetools.com/merchant-center/releases/feed.xml",
    "https://docs.commercetools.com/custom-applications/releases/feed.xml",
    "https://docs.commercetools.com/import-api/releases/feed.xml"
  ]}
/>
```

Reply:
The `title` prop from the RSS feed is a whole sentence...but I was able to extract the feed name from it
d0e13bb

Reply:
The `title` is defined by us here https://github.com/commercetools/commercetools-docs-kit/blob/2678b753d4cc9cbccd344edb6377672c7f51cef9/packages/gatsby-theme-docs/gatsby-config.js#L245

So you should be able to "reverse" that and extract the correct value.

Reply:
You mean by accessing the `siteMetadata`?

Reply:
Like this: https://github.com/commercetools/commercetools-docs-kit/pull/743#discussion_r522085954